cmd/root.go

```diff
@@ -93,4 +98,5 @@ func Execute() {
 func init() {
 	RootCmd.PersistentFlags().StringVarP(&configFile, "config", "c", "checkup.json", "JSON config file")
 	RootCmd.Flags().BoolVar(&storeResults, "store", false, "Store results")
+	RootCmd.Flags().BoolVar(&printLogs, "v", false, "Print logging")
```

Comment:
Change description to "Enable logging to standard output" or something slightly more descriptive